_posts/2018-01-19-event-lineup.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "The 2018 Rust Event Lineup"
+author: Rust Community
+description: "Multitude of Rust event are happening this year; join us at one near you!"
+---
+
+Every year there are multiple Rust events around the world, bringing together the community.
+
+### February 1-3: Mozilla Rust Roadshow Brasil 2018
+
+The [Rust Roadshow][roadshow] is a series of workshops for Rust.
```

Comment:
lets try to get some more info from our friends in Brasil to expand this desc. cc @badboy do you know people from the Latin American Rust Conf planning group that could help?

Reply:
Yep, will contact them.

Reply:
About the Mozilla Rust Roadshow:

The Mozilla Rust Roadshow Brasil is a series of workshops that will happen throughout the country between February 1st and 4th to broaden the adoption of Rust in Brazil. The project is organized by developers and community leaders, contributing along with Mozilla Brasil community organizers.

Find more about who's leading and contributing to the Mozilla Rust Roadshow Brasil 2018: https://mozillabr.org/2018/01/adding-superheroes-to-the-rust-brazilian-community/ 

Learn about the workshops and where they are happening: https://rust-br.github.io/2018-roadshow/

Reply:
Thanks @anapaulavm!